odoo/addons/base/views/ir_module_views.xml

```diff
@@ -68,9 +68,9 @@
                                invisible="not to_buy or state not in ('uninstalled', 'uninstallable')" role="button">Upgrade</a>
                             <button name="button_immediate_upgrade" invisible="state != 'installed'" string="Upgrade" type="object" class="btn btn-primary me-1" groups="base.group_system"/>
                             <button name="button_uninstall_wizard" invisible="state != 'installed'" string="Uninstall" type="object"  class="btn btn-secondary me-1" groups="base.group_system"/>
-                            <button name="button_uninstall_cancel" invisible="state != 'to remove'" string="Cancel Uninstall" type="object" class="btn btn-secondary me-1" groups="base.group_system"/>
-                            <button name="button_upgrade_cancel" invisible="state != 'to upgrade'" string="Cancel Upgrade" type="object" class="btn btn-secondary me-1" groups="base.group_system"/>
-                            <button name="button_install_cancel" invisible="state != 'to install'" string="Cancel Install" type="object" class="btn btn-secondary me-1" groups="base.group_system"/>
+                            <button invisible="state != 'to remove'" class="btn btn-secondary me-1" groups="base.group_system">Removing</button>
+                            <button invisible="state != 'to upgrade'" class="btn btn-secondary me-1" groups="base.group_system">Upgrading</button>
+                            <button invisible="state != 'to install'" class="btn btn-secondary me-1" groups="base.group_system">Installing</button>
```

Comment:
Do I miss something?  It looks like those buttons don't do anything anymore...  Shouldn't they be deactivated?

Reply:
They are `disabled` by the framework automatically (maybe because it is not bound with a rpc?). But it is not bad to explicitly `disable` the button here.
<img width="190" alt="image" src="https://github.com/user-attachments/assets/37cd05a1-1695-4e52-8783-f16087d89596" />